apps/web/content/docs/en/core/accounts.mdx

```diff
@@ -6,45 +6,34 @@ description:
   programs and data accounts. Understand the core concepts of Solana's key-value
   storage system.
 ---
+## Overview
+In the Solana network, all information is stored in **accounts**.
 
-On Solana, all data is stored in what are called "accounts." You can think of
-data on Solana as a public database with a single "Accounts" table, where each
-entry in this table is an "account." Every Solana account shares the same base
-[Account type](https://github.com/anza-xyz/agave/blob/v2.1.13/sdk/account/src/lib.rs#L48-L60).
+You can think of the Solana database as a big table of accounts. Each row in the table is an account with its own address.
+Every account shares the same basic structure (see the struct on the right of the illustration), but stores unique information
+ within that structure.
 
 ![Accounts](/assets/docs/core/accounts/accounts.png)
 
-## Key Points
+## Account address
 
-- Accounts can store up to
-  [10MiB](https://github.com/anza-xyz/agave/blob/v2.1.13/sdk/program/src/system_instruction.rs#L85)
-  of data, which contains either executable program code or program state.
-- Accounts require a
-  [rent deposit](https://github.com/anza-xyz/agave/blob/v2.1.13/sdk/rent/src/lib.rs#L93-L97)
-  in lamports (SOL) that's proportional to the amount of data stored, and you
-  can fully recover it when you close the account.
-- Every account has a program
-  [owner](https://github.com/anza-xyz/agave/blob/v2.1.13/sdk/account/src/lib.rs#L55).
-  Only the program that owns an account can change its data or deduct its
-  lamport balance. But anyone can increase the balance.
-- **Sysvar accounts** are special accounts that store network cluster state.
-- **Program accounts** store the executable code of smart contracts.
-- **Data accounts** are created by programs to store and manage program state.
-
-## Account
-
-Every account on Solana has a unique 32 byte address, often shown as a base58
-encoded string (e.g. `14grJpemFaf88c8tiVb77W7TYg2W3ir6pfkKz3YjhhZ5`).
+In order to pull information from an account, you must be able to call its address. The account address acts as the unique ID 
+for each entry in the "Accounts" table. Every account on Solana has a unique
+32-byte address, often shown as a base58 encoded string (e.g. `14grJpemFaf88c8tiVb77W7TYg2W3ir6pfkKz3YjhhZ5`).
 
 The relationship between the account and its address works like a key-value
-pair, where the address is the key to locate the corresponding on-chain data of
-the account. The account address acts as the "unique ID" for each entry in the
-"Accounts" table.
+pair (in which a key, or unique identifier, gives you access to a particular value, or information). In the Solana
+account model, the "key" is the account address that gives you access to the "value"—the corresponding on-chain information
+in the account. 
 
 ![Account Address](/assets/docs/core/accounts/account-address.svg)
+{/*I personally think this illustration is kinda worthless*/}
```

Comment:
agreed

Reply:
People get protective about their illustrations tho, unfortunately.

For this one, I think it's trying to highlight that the public key is the address of the account. 

My mental image of the blockchain is basically just a looong string of information. Each discrete chunk of information is called an account. Then we have info that points to that information, and those are the account addresses